src/components/form/FluroContentFormField.vue

```diff
@@ -2178,20 +2175,65 @@ export default {
 		dateStringModel: {
 			get() {
 				if (this.timezone) {
```

Comment:
Probably want to verify that the timezone is a valid `moment` timezone - which I think is `moment.zone(this.timezone).isValid()`